src/ingestion/producers/__init__.py

```
@@ -0,0 +1,20 @@
+    run_forever,
+)
+
+__all__ = [
+    "DEFAULT_LINE_STATUS_MODES",
+    "KafkaEventProducer",
+    "KafkaProducerError",
+    "LINE_STATUS_EVENT_TYPE",
+    "LINE_STATUS_POLL_PERIOD_SECONDS",
+    "LineStatusProducer",
+    "run_forever",
+]
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)